### PR TITLE
App polish profile likes

### LINKED ISSUE #699: Missing content type icon under Profile > Likes
All other cards within the app have a fun little icon next to the content type. It appears to be missing within cards under the Profile > Likes cards.

**Steps to Reproduce**
- Visit the Profile section
- Observe a card under the Likes section (Tested with liked content—not the default cards)

**Buggy Behavior**
- Icon is missing next to the content type (Articles, Devotionals, Sermons, etc)

### PR DESCRIPTION
**Pull request by samclaridge**
Fixes #699 #683 and #689

### CODE REVIEW

File: apollos/core/blocks/nav/Link.jsx [hunk 1]
```diff
@@ -74,7 +74,10 @@ export default class NavLink extends Component {
   }
 
   render () {
-    const iconClasses = `${this.props.navItem.icon} display-block`;
+    let icon = this.props.navItem.icon
+    if (this.props.navItem.isActive && this.props.navItem.isActive(this.props) && this.props.navItem.activeIcon) { icon = this.props.navItem.activeIcon }
+
```
Comment (johnthepink) [inline apollos/core/blocks/nav/Link.jsx]: This is a little long to me. Can we add some line breaks and some semicolons? Something like this?

```javascript
let icon = this.props.navItem.icon;
if (
    this.props.navItem.isActive && 
    this.props.navItem.isActive(this.props) && 
    this.props.navItem.activeIcon
) { 
  icon = this.props.navItem.activeIcon;
}
```

**Comment by johnthepink** [comment]
@samclaridge looks good. just one thing <3

**Comment by samclaridge** [comment]
@johnthepink updated per your feedback.

**Comment by johnthepink** [comment]
much nicer!

**Pull request merged**